### PR TITLE
Add 1Password

### PR DESCRIPTION
**Pull request by alanlima**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/572)

### CODE REVIEW

**Comment by galich** [comment]
@alanlima thanks for submitting 1Password package! I'm 1Password for Windows development team lead, one of the checks are failing with

> ##[warning] Manifest Warning: Silent and SilentWithProgress switches are not specified for InstallerType exe. Please make sure the installer can run unattended.

This can be fixed by running `1PasswordSetup-7.4.767.exe --silent`

Do you mind updating your PR with it?

File: manifests/AgileBits/1Password/7.4.767.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Id: AgileBits.1Password
+Version: 7.4.767
+Name: 1Password
+Publisher: AgileBits
+License: AgileBis, Inc. Terms of Service
+LicenseUrl: https://1password.com/legal/terms-of-service/
+AppMoniker: 1password
+Tags: password
+Description: 1Password remembers them all for you. Save your passwords and log in to sites with a single click. It's that simple.
```
Comment (MrRooni) [inline manifests/AgileBits/1Password/7.4.767.yaml:9]: Hey @alanlima , echoing what @galich said, thank you for putting this up. Could you please update this description to the following:

1Password remembers all your passwords for you, and keeps them safe and secure behind the one password that only you know.

**Review by MrRooni — CHANGES_REQUESTED** [review]

**Comment by alanlima** [comment]
Thanks @galich and @MrRooni 😀

I made an amend to includes `--silent` flags as recommended.
I also updated the manifest's description.

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**